### PR TITLE
TACOCAT cladding Properties

### PR DESCRIPTION
**Pull request by thurstonsd**

### CODE REVIEW

**Comment by lcarasik** [comment]
This pull request is intended to partially issue #62 by creating the dictionary for cladding properties.

File: LoadedTACO/src/Clad_Props.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "kclad": 22.2, #Thermal Conductivity averaged between 50C and 1200 C (323.15 K and 1473.15 K) (W/m-k)
+    "Tmelt": 1500.00, #melting Tempurature-C
+    "Tmelt_K": 1773.15, #melting Tempurature-K
+    }
+
+Zirc_2={
+    "kclad": 19.02, #Thermal Conductivity average between 50C and 1000C (323.15 K and 1273.15K)  (W/m-k)
+    "Tmelt":1850.00, #melting Tempurature- C 
+    "Tmelt_K":2123.15, #melting Tempurature-K
+
```
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:22]: Remove one of the extra lines

File: LoadedTACO/src/Clad_Props.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "Tmelt": 1400-1450, #melting Tempurature-C
+    "Tmelt_K": 1673.15-1728.15, #melting tempurature-K
+    }
+
+Zirc_2_Nickel_Free={
+    "kclad":18.1725, #Thermal Conductivity average between 100C and 800C (W/m-k) 
+    }
+
+Graphite_384_1={
+        "kclad":90.016, #Thermal Conductivity average between 101 C and 1000.9 C (374.15 K and 1274.05 K)(W/m-K)
```
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:49]: Fix the indention here for lines 49-51

File: LoadedTACO/src/Clad_Props.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    "Tmelt": 3600, #Melting Point-C
+    "Tmelt_K":3873.15, #Melting Point-K
+    }
+
+Fuel_props = {
+	"Kanth_AMPT": Kanth_AMPT,
+	"Zirc_2": Zirc_2,
+	"Stain_Steel_304": Stain_Steel_304,
+	"Zirc_2_Nickel_Free": Zirc_2_Nickel_Free,
+}
```
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:65]: Fix indentation for lines 61-64

**Review by lcarasik — CHANGES_REQUESTED** [review]
This has been reviewed, there are a few stylistic changes that need to be made before approval.

**Comment by thurstonsd** [comment]
I need to create a new branch on the computer and then send information to github again

**Comment by thurstonsd** [comment]
I need to create a new branch on the computer and then send information to github again

**Pull request closed without merging**